specification/columns/pricingcategory.md

```diff
@@ -2,6 +2,23 @@
 
 Pricing Category describes the pricing model used for a charge at the time of use or purchase. It can be useful for distinguishing between charges incurred at the [*list unit price*](#glossary:list-unit-price) or a reduced price and exposing optimization opportunities, like increasing [*commitment discount*](#glossary:commitment-discount) coverage.
 
+---
+The PricingCategory column adheres to the following requirements:
+
+* PricingCategory MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports more than one pricing category across all SKUs.
+* If present, the column adheres to the following additional requirements:
+  * PricingCategory MUST be of type String.
+  * PricingCategory MUST be null if [ChargeCategory](#chargecategory) is "Tax".
+  * PricingCategory MUST NOT be null if ChargeCategory is "Usage" or "Purchase" and [ChargeClass](#chargeclass) is not "Correction".
+  * PricingCategory MAY be null in all other cases.
+  * If PricingCategory is not null, the following applies:
```

Comment:
```suggestion
  * If PricingCategory is not null, the column adheres to the following additional requirements:
```

Reply:
suggested change applied across the board